meta/autoid/autoid.go

```diff
@@ -293,24 +347,28 @@ func NextStep(curStep int64, consumeDur time.Duration) int64 {
 }
 
 // NewAllocator returns a new auto increment id generator on the store.
-func NewAllocator(store kv.Storage, dbID int64, isUnsigned bool, allocType AllocatorType) Allocator {
+func NewAllocator(store kv.Storage, dbID int64, isUnsigned bool, allocType AllocatorType, info *model.SequenceInfo) Allocator {
```

Comment:
Newallocator() requires a `SequenceInfo` argument? It is weird. 😕
Perhaps it is better to put the sequence allocator as another implementation of `Allocator`.

Reply:
Nice